pkg/ui/karma.conf.js

```diff
@@ -13,6 +13,14 @@ module.exports = function(config) {
     // base path that will be used to resolve all patterns (eg. files, exclude)
     basePath: "",
 
+    // redirect `console.log`s in test code to Karma's stdout.
+    // This is the default behavior in Karma 2.0; so we can remove when we upgrade.
+    browserConsoleLogOptions: {
+      format: "%b %T: %m",
+      level: "log",
+      terminal: true,
+    },
+
```

Comment:
sneaking this in in its own commit. is there an obvious way of debugging without this that I'm missing?

Reply:
this is good thanks